sdk/core/Azure.Core/src/DelayStrategy.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Strategy to control delay behavior.
+    /// </summary>
+    public abstract class DelayStrategy
+    {
+        private readonly Random _random = new ThreadSafeRandom();
+        private readonly double _minJitterFactor;
+        private readonly double _maxJitterFactor;
+        private readonly TimeSpan _maxDelay;
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="maxDelay"></param>
+        /// <param name="minJitterFactor"></param>
+        /// <param name="maxJitterFactor"></param>
+        protected DelayStrategy(TimeSpan? maxDelay = default, double minJitterFactor = 0.8, double maxJitterFactor = 1.2)
```

Comment:
Do you think allowing fine grained control of min and max jitter is needed, or could we simplify to a single `jitterFactor` representing the factor that would be added or subtracted. In this case, we'd default to "0.2" to indicate "20%".

Reply:
I think that is reasonable.